### PR TITLE
[Docs] Button documentation improvement

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
Button documentation improvement - Adding detailed steps for updating text and icon.

## Changes
- Fixes: #5463 

## Preview
https://github.com/cre8ivepark/MixedRealityToolkit-Unity/blob/prerelease/2.1.0_stabilization_Docs_Button/Documentation/README_Button.md#how-to-change-the-icon-and-text

### CODE REVIEW

File: Documentation/README_Button.md [hunk 1]
```diff
@@ -93,6 +93,27 @@ The *ButtonContent* object contains front plate, text label and icon. The *Fron
 
 ![Button](../Documentation/Images/Button/MRTK_Button_Layout.png)
 
+## How to change the icon and text
+
+To change the text of the button, update the *Text* component of the *TextMeshPro* object under *IconAndText*. You can change the icon by replacing the material that is assigned to *UIButtonSquareIcon* object. By default, *HolographicButtonIconFontMaterial* is assigned.
```
Comment (thalbern) [inline Documentation/README_Button.md]:
```suggestion
To change the text of the button, update the *Text* component of the *TextMeshPro* object under *IconAndText*. Changing the icon can be done by replacing the material that is assigned to *UIButtonSquareIcon* object. By default, *HolographicButtonIconFontMaterial* is assigned. 
```

File: Documentation/README_Button.md [hunk 2]
```diff
@@ -93,6 +93,27 @@ The *ButtonContent* object contains front plate, text label and icon. The *Fron
 
 ![Button](../Documentation/Images/Button/MRTK_Button_Layout.png)
 
+## How to change the icon and text
+
+To change the text of the button, update the *Text* component of the *TextMeshPro* object under *IconAndText*. You can change the icon by replacing the material that is assigned to *UIButtonSquareIcon* object. By default, *HolographicButtonIconFontMaterial* is assigned. 
+
+<img src="../Documentation/Images/Button/MRTK_Button_IconUpdate1.png">
+
+To create a new icon material, duplicate one of the existing icon materials. You can find these materials under ``MixedRealityToolkit.SDK/Features/UX/Interactable/Materials`` folder.
```
Comment (thalbern) [inline Documentation/README_Button.md]:
```suggestion
To create a new icon material, duplicate one of the existing icon materials. These can be found under ``MixedRealityToolkit.SDK/Features/UX/Interactable/Materials`` folder. 
```

**Comment by julenka** [comment]
> Having screenshots of property grids usually should be avoided because they tend to change a lot and when updating documentation they get outdated easily and are also forgotten easily. If we can describe things with words (which should be fine how you did it with the paths describing where the materials are and how the properties are called and where to find them) we should stick with that and only add images if they really add information to the docs that can't be easily described in text or are hard to follow in text.

Would be nice to have this in our documentation guidelines / principles so others remember these guidelines as well "Avoid screenshots when decribing property blocks, because they change frequently".

**Comment by cre8ivepark** [comment]
In this case, the screenshots of the Inspector are focusing on the generic 'material' field and the 'albedo' field, I don't expect future changes. But definitely, I agree with the usage of the inspector screenshots!
